common/web/keyboard-processor/src/keyboards/defaultLayouts.ts

```diff
@@ -22,6 +22,7 @@ export type LayoutKey = {
   "nextlayer"?: string,
   "pad"?: string | number,
   "sk"?: LayoutKey[]
+  "default"?: boolean
 }
```

Comment:
For Android:  this (mixed with `ActiveKey`, seen in the file above) is the type of the subkeys provided by Web to the `oskCreatePopup` function found in the keyboard-host page:

https://github.com/keymanapp/keyman/blob/8ba198986fc028f0ff40d5255d3933c5e2cdf67a/android/KMEA/app/src/main/assets/android-host.js#L261-L282

The `obj` parameter is an array of these objects.  Currently, `oskCreatePopup` only references `elementID` (from `ActiveKey`) and `text` - so we'll need something to annotate a 'default' key when calling out from KMW into Android code.

(`elementID` is a mix of the layer ID for the layer hosting the base key, the key's ID [`id`], and the layer ID to use for any relevant key modifiers [`layer`].)